javaScriptFunctions-reusableCode/practiceProblems/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <script src="script1.js"></script>
+</body>
+</html>
```

Comment:
Minor: No new line at EOF, but there's no need to update here. Let's just be mindful of this next time. You can automate this with VS Code.